src/code/z_scene_proc.c

```diff
@@ -113,7 +113,7 @@ void AnimatedMat_DrawTwoTexScroll(PlayState* play, s32 segment, void* params) {
  * Generates a displaylist that sets the prim and env color, and stores it in the provided segment ID.
  */
 void AnimatedMat_SetColor(PlayState* play, s32 segment, F3DPrimColor* primColorResult, F3DEnvColor* envColor) {
-    Gfx* colorDList = GRAPH_ALLOC(play->state.gfxCtx, sizeof(Gfx) * 4);
+    Gfx* colorDList = GRAPH_ALLOC(play->state.gfxCtx, 3 * sizeof(Gfx));
```

Comment:
huh

Reply:
See https://github.com/zeldaret/mm/pull/1137#discussion_r1002562044, this only uses 3 graphics commands, but due to alignment this is the same as allocating for 4 commands.